plugin.xml

```diff
@@ -24,6 +24,10 @@
     <clobbers target="firebase.Analytics" />
   </js-module>
 
+  <js-module src="www/Reference.js" name="reference">
```

Comment:
We should create a global `Database` object on the `firebase` object similar to the [Database](https://developers.google.com/android/reference/com/google/firebase/database/FirebaseDatabase) object in the native api. There are a few methods on it that are generally useful. eg. configuring the getInstance(<config>)  call.

This should be a separate protocol object next to the object representing a `Reference`.